plugins/lua/suspendmanager.lua

```diff
@@ -0,0 +1,130 @@
+local _ENV = mkmodule('plugins.suspendmanager')
+
+local utils = require('utils')
+local gui = require('gui')
+local overlay = require('plugins.overlay')
+local widgets = require('gui.widgets')
+
+--- Loop over all the construction jobs
+---@param fn function A function taking a job as argument
+function foreach_construction_job(fn)
```

Comment:
Who uses this function? Can it be moved to the calling module?

Reply:
Used by `unsuspend` , could be moved there. Although I think that the more reasonable option is to merge `unsuspend` into `suspendmanager`. The widgets can be moved into the Lua module, and the command can be provided by the c++ module. Bonus: the PR to move the rendering of suspension icons to c++ would then become `dfhack` only.

Reply:
Done. :heavy_check_mark: